lumicks/pylake/tests/test_channels/test_channels.py

```diff
@@ -780,6 +799,19 @@ def test_with_data(data, new_data):
     np.testing.assert_allclose(data._with_data(new_data).timestamps, old_timestamps)
 
 
+def test_empty_timeseries():
+    with pytest.raises(IndexError, match="End of empty time series is undefined"):
+        _ = channel.TimeSeries([], []).stop
```

Comment:
is it considered better to use the `_ = ` rather than just nothing at the front?